custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -352,6 +368,10 @@ def _state_aggregation_query(self, state_id):
               AND df.month = %(start_date)s
               AND child_health.state_id = df.state_id
               AND child_health.supervisor_id = df.supervisor_id
+            LEFT OUTER JOIN "{agg_primary_private_school}" ps ON child_health.mother_id = ps.person_case_id
+              AND ps.month = %(start_date)s
+              AND child_health.state_id = ps.state_id
+              AND child_health.supervisor_id = ps.supervisor_id
```

Comment:
how many rows does this return? do we know how adding more joins here will affect performance?